homework-1/css/style.css

```diff
@@ -0,0 +1,1220 @@
+  display: block;
+  width: 24px;
+  height: 18px;
+  position: relative;
+  cursor: pointer;
+}
+.header .menu__icon::before, .header .menu__icon::after, .header .menu__icon span {
+  position: absolute;
+  left: 0;
+  height: 10%;
+  width: 100%;
+  background-color: #fff;
+  transition: all 0.3s ease 0s;
+}
+.header .menu__icon::before, .header .menu__icon::after {
+  content: "";
+}
+.header .menu__icon::before {
+  top: 0;
+}
+.header .menu__icon::after {
+  bottom: 0;
+}
+.header .menu__icon span {
+  top: 50%;
+  transform: scale(1) translate(0px, -50%);
+}
+.header .menu__icon.active span {
+  transform: scale(0) translate(0px, -50%);
+}
+.header .menu__icon.active::before {
+  top: 50%;
+  transform: rotate(-45deg) translate(0px, -50%);
+}
+.header .menu__icon.active::after {
+  bottom: 50%;
+  transform: rotate(45deg) translate(0px, 50%);
+}
+.header .get__link {
+  width: 180px;
+}
+.header .log__link {
+  width: 90px;
+  margin-top: 16px;
+}
+.header__banner {
+  height: 211px;
+}
+.header__banner-title {
+  font-family: "Actor-Regular";
+  font-size: 40px;
+  line-height: 56px;
+  text-align: center;
+  letter-spacing: 2px;
+  text-transform: uppercase;
+  color: #fff;
+  padding-top: 30px;
+  padding: 67px 0 14px;
+  max-width: 142px;
+  margin: 0 auto;
+}
+.header__banner-submenu {
+  padding: 0 20px;
+  position: absolute;
+  width: 100%;
+  top: 198px;
+}
+.first-item {
+  padding-right: 26px;
+  padding-top: 6px;
+}
+.submenu__item-middle {
+  padding-top: 5px;
+}
+.last-item {
+  padding-left: 24px;
+  padding-top: 6px;
+}
+.submenu__list {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 40px;
+  background: #FFFFFF;
+  box-shadow: 0px 12px 24px #F2F2F2;
+  border-radius: 5px;
+  border-top: 2.5px solid #37E63E;
+}
+.submenu__list a {
+  color: #7f7f7f;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 12px;
+  line-height: 20px;
+  text-align: center;
+  letter-spacing: 2px;
+  text-transform: uppercase;
+  transition: 0.5s;
+}
+.submenu__list a:hover {
+  color: #37e63e;
+}
+.submenu__link {
+  border-right: 1.5px solid #e5e5e5;
+  border-left: 1.5px solid #e5e5e5;
+  padding: 0 24px;
+}
+.header__banner {
+  height: 211px;
+}
+.fixed-top {
+  padding: 0 19px;
+  top: 63px;
+}
+#list-top {
+  height: 40px;
+}
+#banner {
+  height: 211px;
+}
+#banner-submenu {
+  padding: 0 19px;
+  top: 62px;
+}
+#header {
+  padding: 18px 29px;
+}
+#header ul {
+  top: -92px;
+}
+.link-hover-center a {
+  display: inline-block;
+  color: #22BF95;
+  line-height: 1;
+  text-decoration: none;
+  cursor: pointer;
+  position: relative;
+}
+.link-hover-center a:after {
+  display: block;
+  content: "";
+  background-color: #22BF95;
+  height: 1.5px;
+  width: 0%;
+  left: 50%;
+  position: absolute;
+  -webkit-transition: width 0.3s ease-in-out;
+  -moz--transition: width 0.3s ease-in-out;
+  transition: width 0.3s ease-in-out;
+  transform: translateX(-50%);
+}
+.link-hover-center.active a:after {
```

Comment:
Before you used some kind of BEM and now you use kebab-case style. Please use only one approach.

Check this through the file

Reply:
fixed a bug